umm-spec-lib/src/cmr/umm_spec/umm_g/granule.clj

```diff
@@ -81,5 +94,12 @@
                             {:EntryTitle entry-title}
                             {:ShortName short-name
                              :Version version-id})
+     :TemporalExtent (if-let [single-date-time (:single-date-time temporal)]
```

Comment:
Is there anything that prevents having both a RangeDateTime and a SingleDateTime in the schema?

Reply:
Yes, it is `one-of`.